src/_pytest/_code/source.py

```diff
@@ -23,7 +24,6 @@ class Source(object):
     def __init__(self, *parts, **kwargs):
         self.lines = lines = []
         de = kwargs.get("deindent", True)
-        rstrip = kwargs.get("rstrip", True)
```

Comment:
i believe this is a breaking api change - what where our guarantees on those internals

Reply:
This isn't an api break, the parameter is ignored (nothing is validated about **kwargs)